f8a_worker/storages/s3_crowd_source_tags.py

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env python3
+
+from . import AmazonS3
+
+
+class S3CrowdSourceTags(AmazonS3):
+
+    def _construct_object_key(self, arguments):
+        path = self.get_object_key_path(arguments['ecosystem'])
+        return "{path}/crowd_sourcing_package_topic.json".format(path=path)
+
+    @staticmethod
+    def get_object_key_path(ecosystem):
+        return "{ecosystem}".format(ecosystem=ecosystem) + "github/data_input_raw_package_list/"
```

Comment:
Shouldn't there be a slash (`/`) between `{ecosystem}` and `"github/data_input_raw_package_list/"` ?

```python
return "{ecosystem}/github/data_input_raw_package_list/".format(ecosystem=ecosystem)
```

Reply:
ping, what about this ?

Reply:
updted